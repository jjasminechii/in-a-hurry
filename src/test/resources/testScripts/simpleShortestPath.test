# Find a path with Dijkstra's algorithm when
# there are two paths possible

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B

AddEdge simpleGraph A B 3.0
AddEdge simpleGraph A B 1.0

FindPath simpleGraph A B