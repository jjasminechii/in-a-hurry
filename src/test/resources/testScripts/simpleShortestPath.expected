# Find a path with Dijkstra's algorithm when
# there are two paths possible

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph

added edge 3.000 from A to B in simpleGraph
added edge 1.000 from A to B in simpleGraph

path from A to B:
A to B with weight 1.000
total cost: 1.000